### PR TITLE
Complete JavaScript arrays exercise. - Dennis Nguyen

### PR DESCRIPTION
**Pull request by dennis-pnguyen**
javascript-arrays

### CODE REVIEW

File: javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+console.log('value of colors:', colors);
+
+const students = [
+    'Tommy',
+    'Chuckie',
+    'Phil',
+    'Lill'
+];
+
+numberOfStudents = students.length;
```
Comment (shawnkost) [inline javascript-arrays/main.js]: You need a `const` before `numberOfStudents`

File: javascript-arrays/main.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    'Chuckie',
+    'Phil',
+    'Lill'
+];
+
+numberOfStudents = students.length; 
+console.log('There are ' + students.length + ' students in the class.');
+
+lastIndex = numberOfStudents - 1; 
+lastStudent = students[lastIndex];
```
Comment (shawnkost) [inline javascript-arrays/main.js]: You need a `const` before these variable declarations

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a couple tweaks here Dennis!

**Comment by dennis-pnguyen** [comment]
javascript-arrays-2
Updated, thank you Shawn!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**